config/schema/field_definitions.json

```diff
@@ -998,5 +998,24 @@
   },
   "digital_market_research_publish_date": {
     "type": "date"
+  },
+
+  "disease_case_closed_date": {
+    "type": "date"
+  },
+  "disease_case_opened_date": {
+    "type": "date"
+  },
```

Comment:
Ditto here as with the other PR alphagov/specialist-publisher#2110

Do we need to define new 'opened' and 'closed' date fields? (I don't know the answer)